atomicfu/src/jvmMain/kotlin/kotlinx/atomicfu/Trace.kt

```diff
@@ -24,41 +24,74 @@ public actual val traceFormatDefault: TraceFormat =
     if (getSystemProperty("kotlinx.atomicfu.trace.thread") != null) TraceFormatThread() else TraceFormat()
 
 private class TraceFormatThread : TraceFormat() {
-    override fun format(index: Int, text: String): String =
+    override fun format(index: Int, text: Any): String =
         "$index: [${Thread.currentThread().name}] $text"
 }
 
 private class NamedTrace(
     private val trace: TraceBase, 
     private val name: String
 ) : TraceBase() {
-    override fun append(text: String) = trace.append("$name.$text")
+    override fun append(arg1: Any) = trace.append("$name.$arg1")
```

Comment:
it would be nice to give these parameters a more meaningful name (in the base class)

Reply:
`event` name probably will do the trick